### ISSUE TITLE
[Suggestion] Double-Chest GUIs should be centred

### ISSUE THREAD

**Issue by ZwipZwapZapony** [jrbudda/Vivecraft_113#21]
If you "use" a chest, its inventory GUI will be displayed above the chest block that you "used". However, this applies to each block of a double-chest separately, meaning that you can get the inventory GUI centred on the left half of the double-chest, or the right half, but not centred on the entire chest (between the two chest blocks).

I would prefer if the inventory GUI of a double-chest is centred above the middle of the whole double-chest, not just above the specific half that you "used". (I'm aware that it may be tough to automatically support random mods' double-chests, as there's probably no clear indication as to what is a stitchable chest and what isn't, but having it for at least the vanilla Minecraft chests would be nice, in my opinion.)

TL;DR: When "using" a double-chest, the GUI will be centred on the half of the chest that you "used". I'd like it to be centred on the entire double-chest instead.
